Dockerfile

```diff
@@ -30,6 +30,7 @@ ENV OPERATOR=/manager \
 
 RUN yum -y install shadow-utils
 RUN useradd -ms /bin/bash nonroot -u 1001
+RUN yum -y update-minimal --security --sec-severity=Important --sec-severity=Critical
```

Comment:
How can we confirm if the security patch has resolved the vulnerability? Or can this be done only during release time?